### ISSUE TITLE
Disable augmentation in training pipeline

### ISSUE THREAD

**Issue by ItsMeTheBee** [tensorflow/models#9142]
# Prerequisites

Please answer the following question for yourself before submitting an issue.

- [ x] I checked to make sure that this issue has not been filed already.

## 1. The entire URL of the documentation with the issue

https://tensorflow-object-detection-api-tutorial.readthedocs.io/en/latest/training.html

## 2. Describe the issue

Hey there!

Not sure if this is a bug because nobody seems to have an issue with this so I guess I just can´t find the right settings.
I followed the linked tutorial to train the SSD ResNet50 V1 FPN 640x640 model with my own dataset (only 1 object).
After monitoring the training with tensorboard I saw that the images used for training are very distorted.
I deleted all the lines responsible for the augmentation but the training images remain distorted.

Images during training look like this
![image](https://user-images.githubusercontent.com/37577432/90988055-0f1f9200-e590-11ea-93bf-5e3a32e60431.png)

The images in my dataset look like this
![image](https://user-images.githubusercontent.com/37577432/90988069-28c0d980-e590-11ea-9d22-2e84dbc995b4.png)

I already asked on stackoverflow but there aren´t many views ans no answers :b
https://stackoverflow.com/questions/63523996/disable-augmentation-in-tensorflow-training-pipeline

So I was wondering how to fix this since I can´t figure out whats going on here.

**Comment by StphMe** [comment]
I have the same issue. Even with no augmentation the images look darker then they should be. I also checked the tf-record files. In them the images are correctly saved.

**Comment by CptK1ng** [comment]
You should normalize your inputs. After normalization between 0 and 1, it should displayed correctly. 

This step is also recommended to improve the learning process. By normalizing inputs, you prevent your training from exploding gradients

**Comment by ymodak** [comment]
For more context also refer this [thread](https://github.com/tensorflow/models/issues/9115#issuecomment-677508264)

**Comment by ItsMeTheBee** [comment]
Do I need to normalize my training dataset or is there a function in the training pipeline i can simply add to achieve this without modifying all of my images??

**Comment by CptK1ng** [comment]
@ItsMeTheBee you can add the following to your training pipeline to rescale all images:

https://www.tensorflow.org/api_docs/python/tf/keras/layers/experimental/preprocessing/Rescaling?hl=de

`inputs = Rescaling(scale=1.0 / 255)(input)`

**Comment by ItsMeTheBee** [comment]
@CptK1ng This is based on training using tf.keras right?
Because tight now I´m only working with the TensorFlow Object Detection API to train networks and I´m not really familiar with using a python script and writing the pipeline myself (but I would be very glad if you could point me towards a tutorial/ repo for EfficientDet or MobileNet to train object detection that uses regular scripts instead of the Object Detection API - I couldn´t find anything that works anymore)

Does adding

```
data_augmentation_options {
    normalize_image {
	original_minval: 0
	original_maxval: 255
	target_minval: 0
	target_maxval: 1
    }
  }
```
in the pipeline achieve the same thing?

**Comment by CptK1ng** [comment]
I have not train a tf2 object detection api model yet, just tf1 versions. Maybe you can try to normalize the image when you create your tf record. Honestly, i have no glue.

**Comment by ItsMeTheBee** [comment]
Hm alright, thanks anyways!

It would be good to know how to configure the training pipeline to fix this.
Maybe @ymodak could help out here?

Apart from that I´m not sure why my training performs so badly if this is just a visualization issue.
I used the same dataset to train a yolo network and it works very well. I converted the dataset to the needed format and checked the boxes with the provided annotation tool and they seem fine so I´m just wondering why the loss looks good but the performance is awful (even if I use images from the training dataset!)

Is there a way to include regular map calculation/ evaluation of the trained network in the pipeline.config?
